vue/test/mixins/locales/locales.test.js

```diff
@@ -0,0 +1,471 @@
+const assert = require("assert");
+const Vue = require("vue").default;
+const vuex = require("vuex");
+
+const { localeMixin } = require("../../../mixins/locales");
+
+describe("Locales Mixin", () => {
+    Vue.use(vuex);
+    this.testComponent = null;
+
+    beforeEach(() => {
+        this.testComponent = new Vue({

```

Comment:
Use a similar approach to https://github.com/ripe-tech/ripe-white/blob/master/vue/test/base.js#L34